### PR TITLE
PDF documentation improvements

### PR DESCRIPTION
**Pull request by MattF-NSIDC**
## Description

- Clean up OBE acknowledgements
- Improve PDF output
    - Nest all `toctree`s under a common header. This is important or the first chapter header gets swallowed and treated as the header for the whole document.
    - Conditional content for HTML site only
    - Add "parts" to the LaTeX / HTML output
    - Tweak LaTeX rendering options: enable pagerefs, URL footers, remove blank pages
- Remove JSON schema output from all doc outputs. It was overwhelming in PDF form, and I don't know how to only display them in HTML cleanly.

## Checklist

If an item on this list is done _or not needed_, simply check it with `[x]`.

- [x] Config lockfile updated (`inv config.export > qgreenland/config/cfg-lock.json`)
- [x] Version bumped if needed (`bumpversion (major|minor|patch|prerelease|build`)
- [x] CHANGELOG.md updated
- [x] Documentation updated if needed
- [x] New unit tests if needed

### CODE REVIEW

File: doc/acknowledgements.md [hunk 1]
```diff
@@ -35,8 +35,6 @@ projects. This is not an exhaustive list -- if your project is listed in our
 * [cmocean](https://github.com/matplotlib/cmocean)
 * [Python](https://www.python.org)
 * [Luigi](https://github.com/spotify/luigi)
-* [Rasterio](https://github.com/mapbox/rasterio)
```
Comment (MattF-NSIDC) [inline doc/acknowledgements.md:38]: We no longer depend on these packages. We may again one day, but for now we do all the computation at the CLI with `gdal`

File: doc/citing.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
+# Citing QGreenland
```
Comment (MattF-NSIDC) [inline doc/citing.md:1]: Extracting some content to individual files made it easier to conditionally include it

File: doc/conf.py [hunk 3]
```diff
@@ -65,7 +65,43 @@
...
+    author,
+    'manual',
+    False,
+)]
+
+latex_logo = '../qgreenland/ancillary/images/qgreenland.png'
+
+latex_show_urls = 'footnote'
+
+latex_show_pagerefs = True
```
Comment (MattF-NSIDC) [inline doc/conf.py:90]: image

File: doc/conf.py [hunk 4]
```diff
@@ -65,7 +65,43 @@
...
+    'QGreenland Documentation',
+    # Authors
+    author,
+    'manual',
+    False,
+)]
+
+latex_logo = '../qgreenland/ancillary/images/qgreenland.png'
+
+latex_show_urls = 'footnote'
```
Comment (MattF-NSIDC) [inline doc/conf.py:88]: image

File: doc/conf.py [hunk 5]
```diff
@@ -65,7 +65,43 @@
...
+
+latex_show_pagerefs = True
+
+latex_elements = {
+    # remove blank pages
+    'classoptions': ',openany,oneside',
+    # Set pdf-wide table-of-contents depth
+    'preamble': r'''
+      \usepackage{hyperref}
+      \setcounter{tocdepth}{3}
```
Comment (MattF-NSIDC) [inline doc/conf.py:98]: Maybe too deep? Go down to `2`?

image

File: doc/index.rst [hunk 6]
```diff
@@ -3,39 +3,67 @@
    You can adapt this file completely to your liking, but it should at least
    contain the root `toctree` directive.
 
+
+========================
+QGreenland Documentation
+========================
```
Comment (MattF-NSIDC) [inline doc/index.rst:9]: This was oddly critical. We struggled for at least an hour with "Why is the first chapter's header getting swallowed, so its children look like chapters?"

**Comment by MattF-NSIDC** [comment]
Looks like ReadTheDocs won't build a PDF on a pull request, only HTML. Oh well, attached
[qgreenland.pdf](https://github.com/nsidc/qgreenland/files/7731676/qgreenland.pdf)
!

File: doc/conf.py (hunk 5 above)
Comment (trey-stafford) [reply doc/conf.py:98]: Hmm, doesn't look too bad to me.

**Review by trey-stafford — APPROVED** [review]

**Pull request merged**